src/SmartComponents/Compliance/ComplianceCard.js

```diff
@@ -78,14 +79,15 @@ const ComplianceCard = ({ fetchCompliance, complianceFetchStatus, complianceSumm
                                                             y: policy.attributes.compliant_host_count
                                                         }, {
                                                             x: 'Non-compliant',
-                                                            y: policy.attributes.total_host_count - policy.attributes.compliant_host_count
-                                                        }
+                                                            y: policy.attributes.test_result_host_count - policy.attributes.compliant_host_count
+                                                        }, (policy.attributes.test_result_host_count === 0 && { x: 'Compliant', y: '0' })
```

Comment:
This seems to override previously defined "Compliant" category. Does this work because of the zero in a string?

Reply:
This is the text shown when you hover over the chart. So when it's 0, it should show `Compliant: 0`. The y values aren't actually required to be unique.